test/unit/sql/data-types.test.js

```diff
@@ -136,12 +146,14 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
 
       testsql('CHAR(12).BINARY', DataTypes.CHAR(12).BINARY, {
         default: 'CHAR(12) BINARY',
+        oracle: 'RAW(12)',
         sqlite: 'CHAR BINARY(12)',
         postgres: 'BYTEA'
       });
 
       testsql('CHAR.BINARY', DataTypes.CHAR.BINARY, {
         default: 'CHAR(255) BINARY',
+        oracle: 'RAW(255)',
```

Comment:
is `RAW` a fixed-length type? If not, this should log a warning (something postgres doesn't do but that is on the "to fix" backlog)

Reply:
Added a warning 'CHAR.BINARY datatype is not of Fixed Length.'